examples/minimal-logs/main.tf

```diff
@@ -0,0 +1,162 @@
+locals {
+  az = "us-east-2a"
```

Comment:
Hardcoding the AZ is not the best, use the datasource for that. 
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones

Reply:
Fixed. Thanks for your input.